src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -172,6 +172,7 @@ const UsersFragment = ({
     ) {
       return (
         <DomainList
+          withKarma={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withKarma` prop is hardcoded to `true`. If this is intentional and you always want to display the karma, then it's fine. However, if you want to make it configurable, consider passing `withKarma` as a prop to the parent component.
```diff
- withKarma={true}
+ withKarma={this.props.withKarma}
```
This change will allow you to control the visibility of the karma score from the parent component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->